language-service/src/utils/yamlServiceUtils.ts

```diff
@@ -1,6 +1,10 @@
 import { getLineOffsets } from "./arrUtils";
 import { TextDocument, Position } from "vscode-languageserver-types";
 
+export const nodeHolder = "h";

```

Comment:
Should we switch this back to `holder` like it was in the original version of the code?  It was reduced to `h` to minimize the bug, but now we have a better fix for it.

Not that it really matters because it's not visible to user, but it might show up error diagnostics.

Reply:
Unfortunately no, it should be of length 1 for my fix to work.

Reply:
I made this comment assuming the conditions above would be loosened.  without that, then yes, it has to stay a single character.